primedev/mods/modmanager.cpp

```diff
@@ -865,17 +865,25 @@ void ModManager::LoadMods()
 				// ensure we're only loading rpaks
 				if (fs::is_regular_file(file) && file.path().extension() == ".rpak")
 				{
-					std::string pakName(file.path().filename().string());
 
+
+					std::string pakName(file.path().filename().string());
```

Comment:
Accidental moving of this line?

Reply:
yeah that mustve happened when i was adding a test logger for something. whoopsies!

Reply:
should be fixed (if vs didnt betray me)